### DISCUSSION TITLE
Bottom Navigation Tab

### DISCUSSION THREAD

**Discussion by tombelieber** [mantinedev/mantine#1213] [Feature requests]
It would be nice to have a bottom navigation tab just like 

https://mui.com/zh/material-ui/react-bottom-navigation
![Screen Shot 2022-04-09 at 11 50 13 PM](https://user-images.githubusercontent.com/47057684/162581460-4cf09708-57f0-41e4-a654-b0667a42d3ee.png)
/

**Comment by clicktodev** [comment]
@rtivital a bottom navigation bar is very handy for mobile navigation.

**Comment by zhulien-ivanov** [comment]
It's literally 3 buttons in a flex container with a fixed position. You don't really need a library component for that.

**Reply by clicktodev** [reply]
I was thinking of an integration with the app shell component.

**Reply by nonameolsson** [reply]
It is not so complicated to build. I agree that a pre-made component would be handy. 

<img width="387" alt="image" src="https://user-images.githubusercontent.com/1493018/193024208-f80843e4-695d-44b7-b9ff-9ad5f220fe2c.png">


This is how I have solved it https://github.com/nonameolsson/linjen/tree/nonameolsson/migrate-to-mantine-194/app/components/bottom-navigation

I created an `Page` component that implements `AppShell`. 
I use it this way https://github.com/nonameolsson/linjen/blob/nonameolsson/migrate-to-mantine-194/app/routes/timeline/%24timelineId.tsx#L107

And this is how I use it to add a bottom navigation
https://github.com/nonameolsson/linjen/blob/nonameolsson/migrate-to-mantine-194/app/components/page.tsx#L90
